package.json

```diff
@@ -559,7 +559,7 @@
     "yargs": "^17.0.1"
   },
   "engines": {
-    "node": "^16.20.0",
+    "node": "^18.17.0",
```

Comment:
Two questions:

* any reason to set v18.17 as a minimum?
* Thoughts on using `>=` instead to allow v20 to be used as well? Or using something like `^18 || ^20`

Reply:
The only reason I did that is it was the version that installed for me when I installed 18 via nvm and it matches the node browsers executor in circleci. I am happy to make it >= 18?

Reply:
would it be `>= 18 || >= 10' ?

Reply:
pushed with '>= 18' but happy to revise

Reply:
Sounds good! The `^18 || ^20` version was for if we wanted to exclude the non-LTS releases, but I think `>=18` makes sense.